### ISSUE TITLE
Permission Requirements to use GetTemplateForItem to get Workflow GUID

### ISSUE THREAD

**Issue by thenoune** [sympmarc/SPServices#57]
I seem to be having an issue with GetTemplateForItem to get a list item workflow GUID.

When I run it with my account, I am able to get the workflow GUID with no issues. When I run it with a regular account, there are no errors returned, but there is also no information about the workflow returned. This is the response text:

   ` <?xml version="1.0" encoding="utf-8"?><soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema"><soap:Body><GetTemplatesForItemResponse xmlns="http://schemas.microsoft.com/sharepoint/soap/workflow/"><GetTemplatesForItemResult><TemplateData><Web Title="TITLE" Url="https://url.com" /><List Title="List Title" Url="https://url.com/Lists/List Title" /><WorkflowTemplates></WorkflowTemplates></TemplateData></GetTemplatesForItemResult></GetTemplatesForItemResponse></soap:Body></soap:Envelope>`

Basically any information about the workflow is not returned. Do you know what permissions are required to use this feature? I've tried giving the user full control over the list item (which doesn't work). The only thing that seems to work is giving more control over the entire site via Sharepoint Site Permissions (which we really don't want to do for everyone!).

It seems like 'Override List Behaviors' and/or 'Enumerate Permissions' via site permissions may allow it to run, but I haven't been able to fully test that yet.

Ideally we wouldn't need to change users full site permissions. I'm assuming there is someplace where we could give users higher permissions on a lower level that doesn't encompass the entire site.

**Comment by sympmarc** [comment]
@thenoune, the permissions would need to be the same as if the user were taking the action using the regular UIs.
